io_gripper_controller/package.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>io_gripper_controller</name>
+  <version>0.0.0</version>
+  <description>gripper io controller used to control the gripper using IOs</description>
+  <maintainer email="yara.shahin@b-robotized.com">Yara Shahin</maintainer>
+  <maintainer email="sachin.kumar@b-robotized.com">Sachin Kumar</maintainer>
```

Comment:
Please add you and Yara as authors and for maintainers, the maintainers of ros2_control. (You can find it in other packages, there are 4 of us).